packages/server/customer-os-api/graph/resolver/dashboard.resolvers_new_customers_it_test.go

```diff
@@ -551,7 +551,7 @@ func TestQueryResolver_Dashboard_New_Customers_MultipleContractsSignedInMonth_Di
 	require.Nil(t, err)
 
 	require.Equal(t, 2, dashboardReport.Dashboard_NewCustomers.ThisMonthCount)
-	require.Equal(t, float64(0), dashboardReport.Dashboard_NewCustomers.ThisMonthIncreasePercentage)
+	require.Equal(t, "+2", dashboardReport.Dashboard_NewCustomers.ThisMonthIncreasePercentage)
```

Comment:
should be +2% ?

Reply:
no, we have an algo for determining that. you can check the tests for
packages/server/customer-os-api/service/dashboard_service_test.go